src/app/tests/suites/certification/Test_TC_DRLK_2_12.yaml

```diff
@@ -0,0 +1,435 @@
+      arguments:
+          values:
+              - name: "Credential"
+                value: { CredentialType: 1, CredentialIndex: 1 }
+      response:
+          values:
+              - name: "CredentialExists"
+                value: true
+              - name: "UserIndex"
+                value: 1
+              - name: "CreatorFabricIndex"
+                value: 1
+              - name: "LastModifiedFabricIndex"
+                value: 1
+              - name: "NextCredentialIndex"
+                value: null
+
+    - label:
+          "TH writes the RequirePINforRemoteOperation attribute value as False
+          on the DUT"
+      PICS: DRLK.S.F07 && DRLK.S.F00 && DRLK.S.A0033
+      command: "writeAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      arguments:
+          value: false
+
+    - label:
+          "TH writes the RequirePINforRemoteOperation attribute value as False
+          on the DUT and Verify DUT responds with UNSUPPORTED_WRITE"
+      PICS: DRLK.S.F07 && DRLK.S.F00 && !DRLK.S.A0033
+      command: "writeAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      arguments:
+          value: false
+      response:
+          error: UNSUPPORTED_WRITE
+
+    - label: "TH reads the RequirePINforRemoteOperation attribute from the DUT"
+      PICS: DRLK.S.F07 && DRLK.S.F00 && DRLK.S.A0033
+      command: "readAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      response:
+          value: false
+
+    - label: "TH sends the Unbolt Door command to the DUT without PINCode"
+      PICS: DRLK.S.C27.Rsp
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 1000
+
+    - label: "TH sends the Unbolt Door command to the DUT with valid PINCode"
+      PICS: DRLK.S.C27.Rsp
+      command: "UnboltDoor"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "PINCode"
+                value: "123456"
+
+    - label:
+          "TH writes the RequirePINforRemoteOperation attribute value as False
+          on the DUT"
+      PICS: DRLK.S.F07 && DRLK.S.F00 && DRLK.S.A0033
+      command: "writeAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      arguments:
+          value: true
+
+    - label:
+          "TH writes the RequirePINforRemoteOperation attribute value as False
+          on the DUT and Verify DUT responds with UNSUPPORTED_WRITE"
+      PICS: DRLK.S.F07 && DRLK.S.F00 && !DRLK.S.A0033
+      command: "writeAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      arguments:
+          value: true
+      response:
+          error: UNSUPPORTED_WRITE
+
+    - label: "TH reads the RequirePINforRemoteOperation attribute from the DUT"
+      PICS: DRLK.S.F07 && DRLK.S.F00 && DRLK.S.A0033
+      command: "readAttribute"
+      attribute: "RequirePINforRemoteOperation"
+      response:
+          value: true
+
+    - label: "TH sends the Unbolt Door command to the DUT with valid PINCode"
```

Comment:
Label does not match test plan, but here the test plan is wrong.  Is that tracked?

Reply:
https://github.com/CHIP-Specifications/chip-test-plans/pull/2801